.travis.yml

```diff
@@ -1,9 +1,36 @@
 dist: bionic
 
+language: python
+
+python:
+  - '3.8'
+
+os: 
+  - linux
+before_install:
+  - sudo apt-get -y install doxygen
+
+install:
+  - pip install -U codespell
+  - pip install -U travis-sphinx
+  - pip install -U sphinx-rtd-theme
+  - pip install -U breathe
+  - pip install -U exhale
+
```

Comment:
This isn't used right now.

Reply:
Not yet, but it will used in a near future

Reply:
> Future work
Unit-Tests
Deploy documentation